react-facet/docs/api/hooks.md

```diff
@@ -43,7 +55,7 @@ const Form = ({ onSubmit, initialValue }) => {
 
   return (
     <fast-div>
-      <fast-input onChange={handleChange} value={value} />
+      <fast-input onKeyUp={handleChange} value={value} />
```

Comment:
`<fast-input />` doesn't accept `onChange` so I switched to `onKeyUp`. Not the same thing but the closest I could come up with